src/Dhadgar.Nodes/README.md

```
@@ -248,279 +120,87 @@ dotnet test tests/Dhadgar.Nodes.Tests --filter "FullyQualifiedName~HelloWorldTes
 
 ### Service Boundaries
 
-The Nodes service follows the microservices pattern established in the Meridian Console platform:
-
 ```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)